docs/detections/rules-ui-create.asciidoc

```diff
@@ -611,10 +616,12 @@ Example using the mustache "current element" notation `{{.}}` to output all the
 === Set up response actions (optional)
 Use Response Actions to set up additional functionality that will run whenever a rule executes.
 
-preview::[]
-
 The Osquery Response Action allows you to include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
 
+The endpoint response action allows you to automatically run response actions on an endpoint when rule conditions are met. Refer to <<isolate-a-host>> to learn more.
+
+IMPORTANT: Host isolation involves quarantining a host from the network to prevent further spread of threats and limit potential damage. Be aware that automatic host isolation from a detection rule execution could result in unintended consequences, such as disrupting legitimate user activities or blocking critical business processes.
```

Comment:
```suggestion
IMPORTANT: Host isolation involves quarantining a host from the network to prevent further spread of threats and limit potential damage. Be aware that automatic host isolation can result in unintended consequences, such as disrupting legitimate user activities or blocking critical business processes.
```
Seemed like potential extra words, feel free to ignore

Reply:
if you like it, also applies in the collapsible block in host-isolation-ov.asciidoc